cypress/integration/attributes.js

```diff
@@ -59,7 +59,7 @@ describe('Attributes test', () => {
         checkNumberOfDots('Resolve', 1)
     })
     it('All picked is displayed', () => {                
-        cy.contains('All picked')
+        cy.get('#attributes').contains('All picked').should('be.visible')
```

Comment:
aha jo, teraz som to chvilu studoval a je to pravda co si hovoril... asi by som to ale riesil v inom PR